src/aui-form-validator/js/aui-form-validator.js

```diff
@@ -424,7 +424,8 @@ var FormValidator = A.Component.create({
          * @default 'a[class=btn-cancel'
          */
         skipValidationTargetSelector: {
-            value: 'a[class~=btn-cancel]'
+            value: 'a[class~=btn-cancel]',
+            value: 'input[type~=checkbox]'
```

Comment:
Hey @orsolyalukacs,

By passing in duplicate `value` key's into the object, I think the previous value gets overridden by the last key passed.  Instead can we pass an array into the `value` attribute and have whatever is calling `instance.get('skipValidationTargetSelector')` to loop over the array values?